datastore/google/cloud/datastore/client.py

```diff
@@ -15,23 +15,21 @@
 
 import os
 
-from google.cloud.datastore_v1.proto import datastore_pb2 as _datastore_pb2
-
 from google.cloud._helpers import _LocalStack
-from google.cloud._helpers import (
-    _determine_default_project as _base_default_project)
+from google.cloud._helpers import (_determine_default_project as
+                                   _base_default_project)
```

Comment:
@chemelnucfin In the future, please don't make cosmetic changes like that that are unrelated to the PR.